### ISSUE TITLE
'minmax(100px, fit-content)' - invalid rule that is not used in test

### ISSUE THREAD

**Issue by tgraham-antenna** [web-platform-tests/wpt#36045]
https://github.com/web-platform-tests/wpt/blob/d75e423ee10689a62795ca7fbcb5c19fa5518df9/css/css-grid/layout-algorithm/grid-intrinsic-size-with-orthogonal-items.html#L23

The test refers instead to a `minmax-auto-fitcontent` that does not exist.

**Comment by Loirooriol** [comment]
`minmax(100px, fit-content)` is not a valid track sizing function anyway...

**Comment by tgraham-antenna** [comment]
Precisely. Whatever someone was trying to test, there's two reasons why it wasn't being tested.
